Engine/about_PSScriptAnalyzer.help.txt

```diff
@@ -0,0 +1,189 @@
+        warnings.  This is likely what generated the most output when you ran the 
+        first time with all the rules enabled.  Now not all of the warnings generated 
+        by the script analyzer are of equal importance.  For the existing script 
+        scenario, try running error and warning rules included but with a few rules 
+        "excluded":
+        
+        PS C:\> $rules = Get-ScriptAnalyzerRule -Severity Error,Warning
+        PS C:\> Invoke-ScriptAnalyzer -Path . -IncludeRule $rules -ExcludeRule `
+                    PSAvoidUsingCmdletAliases, PSAvoidUsingPositionalParameters
+
+        The PSAvoidUsingCmdletAliases and PSAvoidUsingPositionalParameters warnings 
+        are likely to generate prodigious amounts of output.  While these rules have 
+        their reason for being many existing scripts violate these rules over and 
+        over again.  It would be a shame if you let a flood of warnings from these two 
+        rules, dissuade you from looking at potentially more serious warnings.
+        
+        There may be other rules that generate a lot of output that you don't care 
+        about - at least not yet.  As you examine the remaining diagnostics output, 
+        it is often helpful to group output by rule.  You may decide that the one or 
+        two rules generating 80% of the output are rules you don't care about.  You 
+        can get this view of your output easily:
+        
+        PS C:\> $rules = Get-ScriptAnalyzerRule -Severity Error,Warning
+        PS C:\> $res = Invoke-ScriptAnalyzer -Path . -IncludeRule $rules -ExcludeRule `
+                          PSAvoidUsingPositionalParameters, PSAvoidUsingCmdletAliases
+        PS C:\> $res | Group RuleName | Sort Count -Desc | Format-Table Count, Name
+        
+        This renders output like the following:
+        
+        Count Name
+        ----- ----
+           23 PSAvoidUsingInvokeExpression
+            8 PSUseDeclaredVarsMoreThanAssigments
+            8 PSProvideDefaultParameterValue
+            6 PSAvoidUninitializedVariable
+            3 PSPossibleIncorrectComparisonWithNull
+            1 PSAvoidUsingComputerNameHardcoded
+            
+        I may decide to table the PSAvoidUsingInvokeExpression for the moment and 
+        focus on the rest, especially the PSUseDeclaredVarsMoreThanAssigments, 
+        PSAvoidUninitializedVariable and PSPossibleIncorrectComparisonWithNull rules.
+        
+        As you fix rules, go back and enable more rules as you have time to address 
+        the issues.  While getting a completely clean run through every rule is a 
+        noble goal, it may not always be feasible. You have to weigh the gain of
+        passing the rule and eliminating a "potential" issue with changing script
+        and possibly introducing a new problem.  In the end, for existing scripts,
+        it is usually best to have evaluated the rule violations that you deem the 
+        most valuable to address.
```

Comment:
After looking at the comments on splatterbits PR and the sample that is referenced.  I wonder if rule suppression should have its own help topic about_PSScriptAnalyzer_RuleSuppression.help.txt?  One that can show the code samples at https://github.com/PowerShell/PSScriptAnalyzer/blob/master/Tests/Engine/RuleSuppression.ps1  Thoughts?